src/EditorFeatures/Core/GoToDefinition/GoToDefinitionHelpers.cs

```diff
@@ -93,6 +93,51 @@ internal static class GoToDefinitionHelpers
             return definitions.ToImmutableAndFree();
         }
 
+        public static ImmutableArray<DefinitionItem> GetOverridenDefinitions(
+            ISymbol symbol,
+            Project project,
+            bool thirdPartyNavigationAllowed,
+            CancellationToken cancellationToken)
+        {
+            if (symbol is IMethodSymbol method && method.IsOverride)
```

Comment:
why only methods?  Properties/indexers can also have overrides.

Reply:
i believe we have an extension that gets the overridden thing from any symbol.  maybe GetOverriddenSymbol() ?

Reply:
This PR is incomplete. It's a prototype that needs further discussion (agreeing on idea first).